mod/admin.php

```diff
@@ -762,6 +766,39 @@ function admin_page_queue(App $a)
 	]);
 }
 
+/**
+ * @brief Admin Inspect Worker Queue Page
+ *
+ * Generates a page for the admin to have a look into the current queue of
+ * worker jobs. Shown are the parameters for the job and its priority.
+ *
+ * The returned string holds the content of the page.
+ *
+ * @param App $a
+ * @return string
+ */
+function admin_page_workerqueue(App $a)
+{
+	// get content from the queue table
+	$r = q("SELECT `q`.`id`,`q`.`parameter`, `q`.`created`, `q`.`priority`
```

Comment:
We are progressively replacing all deprecated `q()` calls with `dba::select()` calls. In this case, you would have to write:
```php
$statement = dba::select('workerqueue', ['id', 'parameter', 'created', 'priority'], ['done' => 0], ['order'=> ['priority']]);
$r = dba::inArray($statement);
```

I didn't test this, please don't blindly use it.